about.html

```diff
@@ -38,33 +38,75 @@
         <section class="home_section"  id="home">
             <nav class="nav_var_mobile">
                 <img class="burger_icon" src="./assets/icons/burger_icon.png" alt="burger menu" href=""></img>
-                <img class="close_icon inactive" src="./assets/icons/x_icon.svg"></img>
+                <img class="close_icon inactive" src="./assets/icons/x_icon.svg" alt="close icon"></img>
+                <img class="mobile_icon" src="./assets/icons/page_logo.png" alt="close icon"></img>
                 <ul class="nav_var_list_m inactive">
-                    <li class="nav_list_li_m"><a class="nav_list_link_m nav_li" href="">About</a></li>
+                    <li class="nav_list_li_m"><a class="nav_list_link_m nav_li" href="#">About</a></li>
                     <hr class="speaker_line">
-                    <li class="nav_list_li_m"><a class="nav_list_link_p nav_li" href="#program">Program</a></li>
+                    <li class="nav_list_li_m"><a class="nav_list_link_p nav_li" href="index.html#program">Program</a></li>
                     <hr class="speaker_line">
-                    <li class="nav_list_li_m"><a class="nav_list_link_s nav_li" href="#speakers">Speakers</a></li>
+                    <li class="nav_list_li_m"><a class="nav_list_link_s nav_li" href="index.html#speakers">Speakers</a></li>
                     <hr class="speaker_line">
-                    <li class="nav_list_li_m"><a class="nav_list_link_pa nav_li" href="#partners">Sponsors</a></li>
+                    <li class="nav_list_li_m"><a class="nav_list_link_pa nav_li" href="index.html#partners">Sponsors</a></li>
                     <hr class="speaker_line">
                 </ul>
             </nav>
-            <div class="welcome_container">
-                <h1 class="headline_two">WORLD BIOTECHNOLOGY<br>CONFERENCE 2023</h1>
-                <p class="headline_text">A casual scientific meeting to discuss the future of the biotechnology field with ponents from more than 27 countries is taking place in October, In New York.</p>
-                <h2 class="headline_date">2023.10.15(THU) ~ 16(FRI)</h2>
-                <p class="headline_site">@ Grand Palace Hotel, Turistic Center New York</p>
+            <div class="about_container">
+                <h2 class="headline_about">"Hello! Biotech enthusiasts"</h2>
+                <h1 class="about_title">WORLD BIOTECH<br>CONFERENCE 2023</h1>
```

Comment:
- [ ] As this is the main description of your website, it will be better to use a `title` tag, so in this way, it follows the semantic conventions ([title](https://developer.mozilla.org/es/docs/Web/HTML/Element/title#:~:text=El%20elemento%20HTML%20define,dentro%20del%20elemento%20se%20ignoran.&text=Contenido%20de%20metadatos.&text=Texto%20que%20no%20es%20un%20espacio%20en%20blanco%20entre%20elementos.)) of HTML 🟢